src/py42/_internal/clients/alertrules.py

```diff
@@ -0,0 +1,157 @@
+import json
+from py42.clients import BaseClient
+from py42.clients.util import get_all_pages
+from py42.exceptions import Py42NotFoundError
+
+
+_ALERT_RULE_ENDPOINTS = {
+    u"exfiltration": u"query-endpoint-exfiltration-rule",
+    u"cloudshare": u"query-cloud-share-permissions-rule",
+    u"typemismatch": u"query-file-type-mismatch-rule",
+}
+
+
+class AlertRulesClient(BaseClient):
```

Comment:
I am wondering if there are better names for these... Maybe this one should be called `AlertRuleUsersClient` and the other one should be called `AlertRulesClient` ?

Reply:
Does this first client have any other endpoints besides ones for managing users? If not, I think we should make the renaming suggestions (or other suggestions you may have).